src/uu/chcon/src/chcon.rs

```diff
@@ -4,7 +4,7 @@
 
 use clap::builder::ValueParser;
 use uucore::error::{UResult, USimpleError, UUsageError};
-use uucore::format_usage;
+use uucore::{help_about, help_usage};
```

Comment:
please run rustfmt

Reply:
Thank you for the tip !
Just beginning with Rust here learnt something.
Fixed in cf86646